doc/api/child_process.markdown

```diff
@@ -86,7 +86,7 @@ and the `.connected` property is false.
 ### Event: 'message'
 
 * `message` {Object} a parsed JSON object or primitive value
-* `sendHandle` {Handle object} a Socket or Server object
+* `sendHandle` {Handle object} a Socket or Server object, or undefined
```

Comment:
Suggestion: write as `net.Socket` and `net.Server` and make them links to the appropriate sections in net.markdown.  Ditto in process.markdown.